apps/fulfillment-e2e/cypress.ci.config.js

```diff
@@ -0,0 +1,8 @@
+const { defineConfig } = require('cypress');
+const config = require('./cypress.config');
+
+module.exports = defineConfig({
+  ...config,
+  projectId: 'eothcy',
```

Comment:
let's remove this ID for now (I have to create a cypress project and generate new one for FA)

Reply:
Actually, we can remove this file completely now (we will add it when we will integrate these tests is CI)